examples/lightning_logistic_hazard.py

```diff
@@ -0,0 +1,187 @@
+"""A minimal example of how to fit a LogisticHazard model with pytorch lightning
+The point of this example is to make it simple to use the LogisticHazard models in other frameworks
+that are not based on torchtuples.
+
```

Comment:
```suggestion
Original author: Rohan Shad @rohanshad
```

It makes sense to credit this example explicitly to you